src/liquid/components/ld-select/ld-option-internal/ld-option-internal.css

```diff
@@ -99,7 +99,7 @@
     }
   }
 
-  &:where(:not([disabled]):not([aria-disabled='true'])) {
+  &:where(:not(.ld-option-internal--disabled):not([aria-disabled='true'])) {
```

Comment:
I'm not sure if this is needed? As this is an internal component, no one can misuse it...